src/layouts/Layout.astro

```diff
@@ -0,0 +1,173 @@
1-22A28.76 28.76 0 0 1 79 5.13a23.84 23.84 0 0 1 8 5.07 23.2 23.2 0 0 1 5.19 7.59A23.63 23.63 0 0 1 94 27.18a23.63 23.63 0 0 1-1.86 9.39 23.2 23.2 0 0 1-5.19 7.59 23.84 23.84 0 0 1-8 5.07 28.76 28.76 0 0 1-20.19 0 23.4 23.4 0 0 1-14.91-22Zm12.3 0a12.05 12.05 0 0 0 1 5A12.46 12.46 0 0 0 64 38.73a13.32 13.32 0 0 0 9.78 0 12.83 12.83 0 0 0 4.08-2.61 12.36 12.36 0 0 0 2.79-4 12.67 12.67 0 0 0 0-10 12.36 12.36 0 0 0-2.79-4 12.83 12.83 0 0 0-4.08-2.61 13.32 13.32 0 0 0-9.78 0 12.46 12.46 0 0 0-6.81 6.57 12.05 12.05 0 0 0-1.03 5.1ZM101.4 4.56h17.52q6.24 0 9.48 3t3.24 8.64a11.72 11.72 0 0 1-1.23 5.67 10.34 10.34 0 0 1-3.75 3.81 14.21 14.21 0 0 1 4.34 1.47 9.31 9.31 0 0 1 2.88 2.43 9.08 9.08 0 0 1 1.59 3.24 14.85 14.85 0 0 1 .48 3.84 13.58 13.58 0 0 1-1.11 5.64 11.46 11.46 0 0 1-3.12 4.14 13.34 13.34 0 0 1-4.94 2.56 23.13 23.13 0 0 1-6.48.84h-18.9Zm11.76 17.76h2.16q5.52 0 5.52-4.38t-5.52-4.38h-2.16Zm0 18.36h2.58q4.44 0 6.36-1.14a3.94 3.94 0 0 0 1.9-3.66 3.94 3.94 0 0 0-1.92-3.66q-1.92-1.14-6.36-1.14h-2.58ZM170 41.94h-16.8l-2.7 7.86h-12.56l17.22-45.24H168l17.22 45.24h-12.54ZM166.86 33l-5.28-15.06L156.3 33ZM4.74 56.56h17.52q6.24 0 9.48 3T35 68.2a11.72 11.72 0 0 1-1.23 5.67A10.34 10.34 0 0 1 30 77.68a14.21 14.21 0 0 1 4.29 1.47 9.31 9.31 0 0 1 2.88 2.43 9.08 9.08 0 0 1 1.59 3.24 14.85 14.85 0 0 1 .48 3.84 13.58 13.58 0 0 1-1.11 5.64A11.46 11.46 0 0 1 35 98.44a13.34 13.34 0 0 1-4.88 2.56 23.13 23.13 0 0 1-6.48.84H4.74ZM16.5 74.32h2.16q5.52 0 5.52-4.38t-5.52-4.38H16.5Zm0 18.36h2.58q4.44 0 6.36-1.14a4.45 4.45 0 0 0 0-7.32q-1.92-1.14-6.36-1.14H16.5ZM43.86 79.18a23.31 23.31 0 0 1 14.91-22.05 28.76 28.76 0 0 1 20.19 0 23.84 23.84 0 0 1 8 5.07 23.2 23.2 0 0 1 5.19 7.59 24.63 24.63 0 0 1 0 18.78 23.2 23.2 0 0 1-5.19 7.59 23.84 23.84 0 0 1-8 5.07 28.76 28.76 0 0 1-20.19 0 23.4 23.4 0 0 1-14.91-22Zm12.3 0a12.05 12.05 0 0 0 1 5A12.46 12.46 0 0 0 64 90.73a13.32 13.32 0 0 0 9.78 0 12.83 12.83 0 0 0 4.08-2.61 12.36 12.36 0 0 0 2.79-4 12.67 12.67 0 0 0 0-10 12.36 12.36 0 0 0-2.79-4 12.83 12.83 0 0 0-4.08-2.61 13.32 13.32 0 0 0-9.78 0 12.46 12.46 0 0 0-6.81 6.57 12.05 12.05 0 0 0-1.03 5.1ZM127.38 93.94h-16.8l-2.7 7.86H95.34l17.22-45.24h12.84l17.22 45.24h-12.54ZM124.26 85 119 69.94 113.7 85ZM185.4 101.8h-14.64l-11.22-17.4v17.4h-11.76V56.56h18.3a18 18 0 0 1 6.6 1.11 12.43 12.43 0 0 1 7.41 7.47 15.47 15.47 0 0 1 .93 5.4 13.27 13.27 0 0 1-2.49 8.37 12.54 12.54 0 0 1-7.35 4.35Zm-25.86-25h2.22a8.57 8.57 0 0 0 5.34-1.44 5.54 5.54 0 0 0 0-8.28 8.57 8.57 0 0 0-5.34-1.44h-2.22ZM190.32 56.56h17.4a20.52 20.52 0 0 1 8.67 1.86 23.63 23.63 0 0 1 7.08 4.95 23 23 0 0 1 4.77 7.2 22.2 22.2 0 0 1 0 17.16 22.9 22.9 0 0 1-4.71 7.27 23.36 23.36 0 0 1-7.08 5 20.66 20.66 0 0 1-8.73 1.86h-17.4Zm11.76 35.28h2.7a14.61 14.61 0 0 0 5.46-1 11.66 11.66 0 0 0 4-2.64 11.3 11.3 0 0 0 2.52-4 14.28 14.28 0 0 0 .87-5.07 13.92 13.92 0 0 0-.9-5 11.3 11.3 0 0 0-6.6-6.66 14.43 14.43 0 0 0-5.4-1h-2.7Z"></path><circle cx="69.32" cy="77.08" r="19.88"></circle><circle cx="69.32" cy="28.87" r="20.72"></circle></svg></a>
```

Comment:
You can make a text file (`logo.svg` or something), and put in all the content within the `<svg>...</svg>` tags (included). After that, you can directly include the SVG!